### ISSUE TITLE
Support automatic linting of translated content

### ISSUE THREAD

**Issue by stevejalim** [mozilla/bedrock#14948]

**Comment by eemeli** [comment]
Within the next few months, I'll be adding linting/validation support to [moz.l10n](https://github.com/mozilla/moz-l10n/), to replace the current plethora of bespoke L10n linters we use, including e.g. moz-fluent-linter. This will support all of our current formats, including gettext .po files.

It would be very useful for the moz.l10n development work to have Bedrock as an active and interested customer.

**Comment by stevejalim** [comment]
Thanks @eemeli - in reality, I think it'll be a few months until we're putting significant traffic through the CMS<->Smartling integration, so that could fit well. Unless @mathjazz or Team L10N have a strong objection to my suggestion here, I think I'll defer porting any current linter into the Wagtail/Smartling pipeline and wait for the newer linter to land

**Comment by eemeli** [comment]
@stevejalim Would it be possible to get your requirements/expectations enumerated somehow? That would make it much easier to ensure that the moz.l10n linter will serve your needs from the start.

**Comment by stevejalim** [comment]
@eemeli Fair point! TBH I don't have the current behaviour in my head, but I can dig out which linting scripts are currently run in our workflow - will that be enough to help?

**Comment by eemeli** [comment]
From what I could tell, you appear to be using moz-l10n-linter, but if there are others, that would be useful to know. I was actually hoping for more of a description of what sorts of issues you are linting for, what sorts of situations you want a linter for, how you use the validation results, and how configurable all of this needs to be.

But if the ask is just, "something that works", then we can aim for that too and use our own judgement.
